src/components/style/BlogSingle.css

```diff
@@ -0,0 +1,31 @@
+.BlogSingle-section {
+  padding-top: 112px;
+  padding-bottom: 112px;
+  display: grid;
+  grid-template-columns: 1fr 404px;
+}
+
+.BlogSingle-ft-img img {
+  width: 100%;
+  height: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-bottom: 48px;
+}
+.BlogSingle-content h2 {
```

Comment:
It is not suggested to apply style on a tag `eg: h2, p, etc...`
Try using camelCase as well in your naming. eg `blogSingle-content` instead of `BlogSingle-content`

Reply:
Well done sir.